crates/core/src/client/mod.rs

```diff
@@ -315,7 +315,7 @@ impl<P: Provider + Send + Sync> KakarotEthApi<P> for KakarotClient<P> {
 
                     TransactionReceipt {
                         transaction_hash,
-                        transaction_index: None,
+                        transaction_index: Some(U256::ZERO),
```

Comment:
yep*. not sure if its a good move to merge these changes until we are out of an explorational phase. could be overfitting to just forge.

<details>
*it will be on both madara and katana, as it relates to how the ethers client parses our response. 
</details>